### DISCUSSION TITLE
CurvedNess

### DISCUSSION THREAD

**Discussion by joshemmitt** [cnr-isti-vclab/meshlab#1054] [Q&A]
Under "Compute curvature principal directions" there is a a quality/color mapping option called "CurvedNess". I cannot find any documentation on this algorithm, does anyone know what it does/how it measures curvedness (including what the output values mean)? Or point me towards a publication for the measure.

Thank you for any help

**Comment by alemuntoni** [comment]
Hi, 
when selecting "CurvedNess", [this function of the vcg library](https://github.com/cnr-isti-vclab/vcglib/blob/8132239bfa5bc864a016b4fb3a27f389e4921ca4/vcg/complex/algorithms/update/quality.h#L334) is called.
Documentation of this function says:
```
 Compute from the current Curvature Direction the Curvedness as defined by [Koenderink 1992]
 and store it in the per-vertex Quality.
 C =  Sqrt((k1*k1+k2*k2)/2.0)
 
 J. Koenderink and A. van Doorn. 
 Surface shape and curvature scales. 
 Image and vision computing, 10(8):557–565, 1992.
```

Hope this helps!
